samples/api/hello_triangle/hello_triangle.cpp

```diff
@@ -435,24 +435,7 @@ void HelloTriangle::init_swapchain(Context &context)
 	VkSurfaceCapabilitiesKHR surface_properties;
 	VK_CHECK(vkGetPhysicalDeviceSurfaceCapabilitiesKHR(context.gpu, context.surface, &surface_properties));
 
-	uint32_t surface_format_count;
-	vkGetPhysicalDeviceSurfaceFormatsKHR(context.gpu, context.surface, &surface_format_count, nullptr);
-	std::vector<VkSurfaceFormatKHR> supported_surface_formats(surface_format_count);
-	vkGetPhysicalDeviceSurfaceFormatsKHR(context.gpu, context.surface, &surface_format_count, supported_surface_formats.data());
-
-	// We want to get an SRGB image format that matches our list of preferred format candidates
-	// We initialize to the first supported format, which will be the fallback in case none of the preferred formats is available
-	VkSurfaceFormatKHR format                = supported_surface_formats[0];
-	auto               preferred_format_list = std::vector<VkFormat>{VK_FORMAT_R8G8B8A8_SRGB, VK_FORMAT_B8G8R8A8_SRGB, VK_FORMAT_A8B8G8R8_SRGB_PACK32};
-
-	for (auto &candidate : supported_surface_formats)
-	{
-		if (std::find(preferred_format_list.begin(), preferred_format_list.end(), candidate.format) != preferred_format_list.end())
-		{
-			format = candidate;
-			break;
-		}
-	}
+	VkSurfaceFormatKHR format = vkb::select_surface_format(context.gpu, context.surface);
```

Comment:
Note sure if we should remove this part. The hello_triangle samples should is intended to be as explicit as possible, so it's probably better to put the whole code from `vkb::select_surface_format` in here.

Reply:
As per today's call it's fine if the function is used here.